CHANGELOG_INTERNAL.md

```diff
@@ -4,6 +4,9 @@ This file contains all the changes in the **internal bundle** (used by Builder).
 
 All the changes that affects the public bundle should be released as *patch* or *minor* and be included in the main Changelog.
 
+## 4.1.12-0 - 2019-08-02
+- Change attribution character and toggle widget based on container [#2235](https://github.com/CartoDB/carto.js/pull/2235)
```

Comment:
I think it would be `4.1.11-1` as we haven't released any `4.1.11` public version